src/Pretty.hs

```diff
@@ -309,8 +322,8 @@ instance Pretty VarInfo (Scheme Ty) where
     pp env t
   pp env (Scheme argKinds t) =
     text "∀" <>
```

Comment:
This should prevent many of the confusing newline issues in the tests:
```suggestion
   group $ text "∀" <>
```

Lining up the bindings is nice - but shouldn't they also get grouped, so we can get
```
forall (x : A) (y : B). C
```
instead of always breaking lines?